python/llm/src/ipex_llm/transformers/npu_models/baichuan_mp.py

```diff
@@ -111,7 +112,7 @@ def run_model(
     return results
 
 
-class LowBitLlamaMultiDecoderlayer(NNFactory):
+class LowBitLlamaMultiDecoderlayer(LLMBaseNNFactory):
```

Comment:
maybe we modify to `LowBitBaichuanMultiDecoderlayer`, other classes also.

Reply:
updated